File: haskell/iniciante/1046.hs
-- title: Tempo de Jogo
-- number of the problem: 1046
-- date: 22/09/20
-- author: ltsdw

main :: IO ()
main = do
    l1 <- getLine

    let (start : end : _) = map (read :: String -> Int) $ words l1

    if start == end
        then putStrLn "O JOGO DUROU 24 HORA(S)"
    else if start > end
        then putStrLn $ "O JOGO DUROU " ++ show (24 - start + end) ++ " HORA(S)"
    else
        putStrLn $ "O JOGO DUROU " ++ show (end - start) ++ " HORA(S)"

File: python/iniciante/1046.py
'''
title: Tempo de Jogo
number of the problem: 1046
date: 05/03/2018
author: ltsdw
'''

c1, c2 = map(int, input('').split())

cn = 0

if c2 > c1:
    for c1 in range(c1, c2):
        cn += 1
    print(f'O JOGO DUROU {cn} HORA(S)')
else:
    for c1 in range(c1, 24):
        c2 += 1
    print(f'O JOGO DUROU {c2} HORA(S)')

